### PR TITLE
chore(docs): add developer handbook

### PR DESCRIPTION
**Pull request by pihme**
Adds developer handbook

Depends in part on #161

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,236 +38,247 @@ The different layers work well together, but can also be called directly. E.g. i
...
+| Delete Generation in Camunda Cloud                | `delete-generation-in-camunda-cloud` / `delete-generation-in-camunda-cloud-job`                               | `generationUUID`                                                                                                           |                                                                                                                    |     |
+| Run Chaos Experiments                             | `run-chaos-experiments` / `chaos-experiments`                                                                 | `authenticationDetails`, `clusterPlan`                                                                                     | `testResult`, `testReport`                                                                                         |
```
Comment (Zelldon) [inline README.md:277]: I think you broke the table

File: docs/developer-handbook.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Common Tasks
+
+### Adding Workers
+
+- When you add Java workers as part of the core code and testbench deployment, then make sure to register the workers in the launcher
+- When you add workers that are to be run as a separate deployment, make sure to update `ci/scripts/deploy.sh` in three places
+  - Deploy the pod
+  - Restart the pod (see comment in file on why this is necessary)
+  - Wait for the pod
```
Comment (Zelldon) [inline docs/developer-handbook.md:39]: Should we add that shell workers should be validated via shellcheck?

File: docs/developer-handbook.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+- Add the secret to the `deploy.sh` script
+
+```Bash
+kubectl create secret generic testbench-secrets --namespace="${namespace}" \
+  --from-literal=contactPoint="${CONTACT_POINT}" \
+  --from-literal=clientSecret="${CLIENT_SECRET}" \
+```
+
+- Add the secret to the deployment arteficts (one for dev and one for int) where you want to use it
```
Comment (Zelldon) [inline docs/developer-handbook.md:64]:
```suggestion
- Add the secret to the deployment artefacts (one for dev and one for int) where you want to use it
```

**Review by Zelldon — APPROVED** [review]
Thanks :+1: just small note

File: README.md (hunk 1 above)
Comment (pihme) [reply README.md:277]: Interesting. Must have been the editor doing this automatically. It also aligned the columns

**Pull request merged**